x-pack/plugins/security_solution/common/search_strategy/security_solution/risk_score/common/index.ts

```diff
@@ -23,21 +23,23 @@ export const getHostRiskIndex = (
   onlyLatest: boolean = true,
   isNewRiskScoreModuleInstalled: boolean
 ): string => {
-  return isNewRiskScoreModuleInstalled
-    ? getRiskScoreLatestIndex(spaceId)
-    : `${RISKY_HOSTS_INDEX_PREFIX}${onlyLatest ? 'latest_' : ''}${spaceId}`;
+  if (isNewRiskScoreModuleInstalled) {
+    return onlyLatest ? getRiskScoreLatestIndex(spaceId) : getRiskScoreTimeSeriesIndex(spaceId);
+  } else {
+    return `${RISKY_HOSTS_INDEX_PREFIX}${onlyLatest ? 'latest_' : ''}${spaceId}`;
+  }
 };
 
 export const getUserRiskIndex = (
   spaceId: string,
   onlyLatest: boolean = true,
   isNewRiskScoreModuleInstalled: boolean
 ): string => {
-  return isNewRiskScoreModuleInstalled
-    ? onlyLatest
-      ? getRiskScoreLatestIndex(spaceId)
-      : getRiskScoreTimeSeriesIndex(spaceId)
-    : `${RISKY_USERS_INDEX_PREFIX}${onlyLatest ? 'latest_' : ''}${spaceId}`;
+  if (isNewRiskScoreModuleInstalled) {
+    return onlyLatest ? getRiskScoreLatestIndex(spaceId) : getRiskScoreTimeSeriesIndex(spaceId);
+  } else {
+    return `${RISKY_USERS_INDEX_PREFIX}${onlyLatest ? 'latest_' : ''}${spaceId}`;
+  }
```

Comment:
NIT: this `getUserRiskIndex` implementation is 99% the same as `getHostRiskIndex`. Maybe it's worth extracting this logic into a `getEntityRiskIndex`, receiving the index prefix (the only thing that changes), this way we keep the code DRY.